holmes/plugins/toolsets/infrainsights/base_toolset.py

```
@@ -0,0 +1,181 @@
+"""
+Base Toolset for InfraInsights Toolsets
+
+This provides common functionality for all InfraInsights toolsets including
+instance discovery, credential management, and user context handling.
+"""
+
+import logging
```

Ruff (0.12.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)